e2e_test/udf/general/create_and_drop.slt

```diff
@@ -93,23 +102,5 @@ drop aggregate mysum;
 statement ok
 drop aggregate if exists mysum;
 
-statement ok
-create function add(a int, b int) returns int language python as $$
-def add(a, b):
-  return a+b
-$$;
-
-statement ok
-create sink s as select add(a, b) as c from t with (connector = 'blackhole');
```

Comment:
This will panic if put it after dropping MVs above. See https://github.com/risingwavelabs/risingwave/issues/20527